docs/guides/lint-arazzo.md

```diff
@@ -120,6 +120,16 @@ To expand the linting checks for an Arazzo description, start by enabling
 some of the built-in rules. The currently-supported rules are:
 
 - `parameters-no-body-inside-in`: the `in` section inside `parameters` must not contain a `body`.
+- `strict-source-description-type`: the `type` property of the `sourceDescription` object must be either `openapi` or `arazzo`.
+- `arazzo-version-enum`: the `version` property must be one of the allowed values: [`1.0.0`].
+- `workflow-workflowId-unique`: the `workflowId` property must be unique across all workflows.
```

Comment:
```suggestion
- `workflowId-unique`: the `workflowId` property must be unique across all workflows.
```

What about removing the duplication?

Reply:
its not duplication, but rather describe nested structure `workflow -> workflowId-unique`

Reply:
It makes a lot of sense! But I can't see another place where `stepId` or `workflowId` are used, so I think we could abbreviate it to make it more readable.

Reply:
Renamed, thx